bot.py

```diff
@@ -385,6 +385,8 @@ def check(m):
         if len(session[1]) == 0:
             await client.change_presence(game=client.get_server(WEREWOLF_SERVER).me.game, status=discord.Status.online)
     await send_lobby(leave_msg)
+    if member in list(session[1]) and session[0]:
```

Comment:
What's the purpose of this? Member isn't even defined outside of the for loops and each member would have already been removed from the game on line 382